draft-ietf-rtcweb-jsep.xml

```diff
@@ -3642,27 +3642,34 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         processing MUST stop and an error MUST be returned.
         <list style="symbols">
 
-          <t>If the PeerConnection state is "have-local-offer", and the
-          ICE ufrag or password changed from the previous remote
+          <t>If the ICE ufrag or password are unchanged from the previous remote description,
```

Comment:
I read this a few times, and it's technically correct, but it's kind of hard to parse.

I sort of prefer the original outline from the bug which was easier to follow. Was there a reason you chose this different structure?